cmd/kube-apiserver/app/options/plugins.go

```diff
@@ -83,4 +90,46 @@ func RegisterAllAdmissionPlugins(plugins *admission.Plugins) {
 	setdefault.Register(plugins)
 	resize.Register(plugins)
 	pvcprotection.Register(plugins)
+
+	// Set recommended plugins order
+	recommendedOrder := []string{
+		deny.PluginName,                       // AlwaysDeny
```

Comment:
putting this last will still fail, but will allow you to execute the rest of the chain for testing purposes.

Reply:
I want to fail fast. Actually, this is no use, no one will enable deny with other plugins simultaneously.

Reply:
> I want to fail fast. Actually, this is no use, no one will enable deny with other plugins simultaneously.

I don't think that's true.  "I want to test admission configuration without accidentally creating any resources".  In such a case, this must come last.

I don't see a similar argument for having it first.

Reply:
I see. This use case is reasonable.